src/index.html

```diff
@@ -11,7 +11,208 @@
+            <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-input">
+          <label for="age">
+            How old are you?
+            <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            >
+          </label>
+        </div>
+
+        <div class="form-input">
+          <label for="date">
+            Full date of birth
+            <input
+            type="date"
+            name="date"
+            id="date"
+            >
+          </label>
+        </div>
+
+        <div class="form-input check">
+          <label for="check">
+            I accept the term of agreement
+            <input
+            type="checkbox"
+            name="check"
+            id="check"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+
+        <div class="form-input">
+          <label for="email">
+            Email:
+            <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="form-input password">
+          <label for="password">
+            Password:
+            <input
+            type="password"
+            name="password"
+            id="password"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-input">
+          Do you love cats?
+
+          <label for="cats">
+            <input
+            type="radio"
+            name="cats"
+            value="yes"
+            >
+            Yes
+          </label>
+
+          <label for="cats">
+            <input
+            type="radio"
+            name="cats"
+            value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-input">
+          <label for="color">
+            What is your favorite color?
+            <input
+            type="color"
+            name="color"
+            id="color"
+            >
+          </label>
+        </div>
+
+        <div class="form-input">
+          <label for="time">
+            What time do you go to bed?
+            <input
+            type="time"
+            name="time"
+            id="time"
+            >
+          </label>
+        </div>
+
+        <div class="form-input">
+          <label for="cars">
+            What are your favorite brand of cars?
+            <textarea
+            name="cars"
+            id="cars"
+            cols="5"
+            rows="4"
+          >BMW Audi Lada
+            </textarea>
```

Comment:
It's should be a select field with `multiple `attribute options instead of textarea